test/usage.js

```diff
@@ -874,6 +877,52 @@ describe('usage tests', function () {
       r.logs[0].should.eql('1.0.1')
     })
 
+    describe('default version #', function () {
+      before(function (done) {
+        this.timeout(10000)
+        // create a symlinked, and a physical copy of yargs in
+        // our fixtures directory, so that we can test that the
+        // nearest package.json is appropriately loaded.
+        cpr('./', './test/fixtures/yargs', {
+          filter: /node_modules|example|test|package\.json/
```

Comment:
To make this more of a "real world" scenario, we probably shouldn't filter out `package.json` (since yargs will have one when installed as a dependency). Perhaps instead we should copy a control `package.json` file with a constant version so we can guarantee its version is not picked up by the test.